### PR TITLE
Clarification of delegating and staking

### PR DESCRIPTION
**Pull request by timothymcmackin**
Preview: https://docs-staging-git-staking-delegating-trili-tech.vercel.app/architecture/stakers

### CODE REVIEW

**Comment by timothymcmackin** [comment]
Closing in favor of https://github.com/trilitech/tezos-developer-docs/pull/463.

File: docs/overview/tezos-different.md [hunk 1]
```diff
@@ -20,11 +20,11 @@ Anyone who holds XTZ — the chain's native token — can propose changes to how
...
+The proof-of-stake consensus model eliminates the need for high energy use, making it the "green" choice for blockchains. Instead of competing to achieve consensus as in proof-of-work models, Tezos nodes (called *bakers*) stake Tezos tokens to earn the right to create blocks and receive rewards. Users who want to participate without running a node themselves can delegate and stake tokens to a baker for a share of the rewards. The bakers keep control of their tokens in that they can unstake them later; and delegators keep complete control of their tokens. Tezos's approach to consensus has been described as [Liquid Proof of Stake](https://medium.com/tezos/liquid-proof-of-stake-aec2f7ef1da7).
```
Comment (NicNomadic) [inline docs/overview/tezos-different.md:23]: What happens to these changes after the PR has been closed, will they be redone in another PR?
